src/living/helpers/gui2d/control.ts

```diff
@@ -164,12 +163,13 @@ export class Control2D {
   private _lastRect: DOMRectReadOnlyImpl;
   private _lastCursor: BABYLON.Vector2;
   private _isCursorInside = false;
-  private _renderingContext: CanvasRenderingContext2D;
+  protected _renderingContext: CanvasRenderingContext2D; // for unit test
   private _overwriteHeight: number;
   private _overwriteWidth: number;
-  private _imageData: ImageDataImpl;
+  private _imageData: ImageBitmap;
```

Comment:
1. Private field should be a *impl type
2. The name should be `_imageBitmap` if its type is `ImageBitmap*`